api/src/pcapi/core/offerers/api.py

```diff
@@ -176,20 +177,19 @@ def update_venue(
         venue_snapshot.trace_update(contact_data.dict(), target=target, field_name_template="contact.{}")
         upsert_venue_contact(venue, contact_data)
 
-    for daily_opening_hours in opening_hours or []:
-        weekday = models.Weekday(daily_opening_hours.weekday)
-        target = get_venue_opening_hours_by_weekday(venue, weekday)
-        target.timespan = date_utils.numranges_to_readble_str(target.timespan)
-        opening_hours_readable = {
-            "weekday": daily_opening_hours.weekday,
-            "timespan": date_utils.numranges_to_readble_str(daily_opening_hours.timespan),
+    if opening_hours:
+        old = opening_hours_api.old_opening_hours(venue)
```

Comment:
Détail, low logaf

Je préfère le naming `current` que `old` ici

Par exemple
```suggestion
        old = opening_hours_api.get_current_opening_hours(venue)
```

ça me parait plus proche de ce que fait la fonction